requirements.txt

```diff
@@ -18,6 +18,7 @@ netifaces
 numba
 numpy
 pyephem
+pyjwt==1.7.1
```

Comment:
Could you add this to katsdpdockerbase - we're going to need to use it in all packages that use katdal.

Reply:
Can I first add it to katsdpdockerbase, and then come back and remove the pin here? Does the order matter?

Reply:
If you remove it here before adding it to katsdpdockerbase, the build of this will fail. So I'd recommend adding it to katsdpdockerbase first.

Reply:
Done.